prov/efa/src/efa_domain.c

```diff
@@ -210,9 +211,14 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 		return -FI_EINVAL;
 	}
 
-	/* Check the value of environment variable FI_EFA_USE_DEVICE_RDMA */
-	efa_domain->use_device_rdma = rxr_env_get_use_device_rdma();
```

Comment:
I think we should move`use_device_rdma` to inside `rxr_ep`.

Reply:
It is.  I just cached it to strictly replicate pre-1.18 API behavior.  My latest push removes that cache and the value lives only in the rxr_ep.